### DISCUSSION TITLE
Repository link

### DISCUSSION THREAD

**Discussion by Nikitha1612** [community/community#55071] [New to GitHub]
I have created a repository and upload a jupyter notebook . I tried to generate live link in settings and I changed the branch to main but still I could not get link. Is there any solution to this issue?

**Comment by SoundaryaKoutharapu** [comment]
You can store any code you like in a GitHub repository, but to use the GitHub Pages feature to full effect, 
your code should be structured as a typical website, e.g. with the primary entry point being an HTML file called index. html
